webapp/src/Entity/User.php

```diff
@@ -436,22 +436,21 @@ public function getRoleList(): array
     #[Serializer\Type('string')]
     public function getType(): ?string
     {
-        // Types allowed by the CCS Specs Contest API in order of most permissions to least.
+        // Types suggested by the CCS Specs Contest API in order of most permissions to least.
         // Either key=>value where key is the DOMjudge role and value is the API account type or
         // only value, where both the DOMjudge role and API type are the same.
-        $allowedTypes = ['admin', 'api_writer' => 'admin', 'api_reader' => 'admin',
-                         'jury' => 'judge', 'api_source_reader' => 'judge',
-                         'team'];
-        foreach ($allowedTypes as $role => $allowedType) {
+        $mappedTypes = ['admin', 'api_writer' => 'admin', 'api_reader' => 'admin',
+                        'jury' => 'judge', 'api_source_reader' => 'judge', 'team'];
+        foreach ($mappedTypes as $role => $mappedType) {
             if (is_numeric($role)) {
-                $role = $allowedType;
+                $role = $mappedType;
             }
             if (in_array($role, $this->getRoleList())) {
-                return $allowedType;
+                return $mappedType;
             }
         }
 
-        return null;
+        return 'other';
```

Comment:
```suggestion
        return 'domjudge_internal';
```
So other systems reading it know who to contact if they want to do anything with the account.

Reply:
I'm not strongly against this, but why is this better than just `other`? If you're querying the DOMjudge API, you know the source already. Do we want to distinguish our nonstandard accounts from possibly another system's nonstandard accounts?x

Reply:
The CDS might know the source, but things behind the CDS don't, if you make contest exports it's more convenient etc.